src/App.tsx

```diff
@@ -14,7 +14,9 @@ export default function App() {
     <ApolloProvider client={apolloClient}>
       <PersonalDataProvider>
         <OverlayManagerProvider>
-          <LayoutWithRouting />
+          <DraftsProvider>
```

Comment:
This provider should probably be wrapping studio routes, we don't need that in the regular consumer experience. You will need to rebase off the latest changes